lib/pages/chat_list/chat_list_item.dart

```diff
@@ -143,42 +143,46 @@ class ChatListItem extends StatelessWidget {
-          children: [
-            ListTile(
-              visualDensity: const VisualDensity(vertical: -0.5),
-              contentPadding: const EdgeInsets.symmetric(horizontal: 8),
-              onLongPress: onLongPress,
-              leading: selected
-                  ? SizedBox(
-                      width: AvatarStyle.defaultSize,
-                      height: AvatarStyle.defaultSize,
-                      child: Material(
-                        color: Theme.of(context).primaryColor,
-                        borderRadius: BorderRadius.circular(AvatarStyle.defaultSize),
-                        child: const Icon(Icons.check, color: Colors.white),
-                      ),
-                    )
-                  : Avatar(
-                      mxContent: room.avatar,
-                      name: displayname,
-                      onTap: onLongPress,
-                    ),
-              title: Row(
-                children: <Widget>[
-                  Expanded(
-                    child: Row(
+        child: ListTile(
+          visualDensity: const VisualDensity(vertical: -0.5),
+          contentPadding: const EdgeInsets.symmetric(horizontal: 8),
+          onLongPress: onLongPress,
+          leading: selected
+              ? SizedBox(
+                  width: AvatarStyle.defaultSize,
+                  height: AvatarStyle.defaultSize,
+                  child: Material(
+                    color: Theme.of(context).primaryColor,
+                    borderRadius: BorderRadius.circular(AvatarStyle.defaultSize),
+                    child: const Icon(Icons.check, color: Colors.white),
+                  ),
+                )
+              : Avatar(
+                  mxContent: room.avatar,
+                  name: displayname,
+                  onTap: onLongPress,
+                ),
+          title: Row(
+            children: <Widget>[
+              Expanded(
+                child: Column(
+                  children: [
+                    Row(
                       children: [
                         Flexible(
                           child: Text(
                             displayname,
                             overflow: TextOverflow.ellipsis,
                             maxLines: 1,
                             softWrap: false,
-                            style: Theme.of(context).textTheme.titleMedium
-                                ?.merge(const TextStyle(
-                                  overflow: TextOverflow.ellipsis, 
-                                  letterSpacing: 0.15)),
+                            style: Theme.of(context).textTheme.titleMedium?.merge(
+                                  TextStyle(
+                                    overflow: TextOverflow.ellipsis,
+                                    letterSpacing: 0.15,
+                                    color:
+                                        unread ? const Color(0xFF1C1B1F) : const Color(0xFF787579),
```

Comment:
You should use Theme.of to look up color as much as possible, don't use raw Color, in this case please use: `Theme.of(context).colorScheme.onSurface`